meilisearch-http/src/routes/indexes/documents.rs

```diff
@@ -296,8 +300,13 @@ pub async fn delete_documents(
     index_scheduler: GuardedData<ActionPolicy<{ actions::DOCUMENTS_DELETE }>, Data<IndexScheduler>>,
     path: web::Path<String>,
     body: web::Json<Vec<Value>>,
+    req: HttpRequest,
+    analytics: web::Data<dyn Analytics>,
 ) -> Result<HttpResponse, ResponseError> {
     debug!("called with params: {:?}", body);
```

Comment:
Why do we have a debug there?

Reply:
We're supposed to have this debug on _all_ the routes :grin: